assets/src/edit-story/components/panels/design/border/borderWidth.js

```diff
@@ -176,25 +177,27 @@ function WidthControls({ selectedElements, pushUpdateForObject }) {
         </>
       )}
       <ToggleWrapper locked={lockBorder}>
-        <LockToggle
-          isLocked={lockBorder}
-          onClick={() => {
-            let args = {
-              lockedWidth: !lockBorder,
-            };
-            // If the border width wasn't locked before (and is now), unify all the values.
-            if (!lockBorder) {
-              args = {
-                ...args,
-                top: border.left,
-                right: border.left,
-                bottom: border.left,
+        <Tooltip title={__('Toggle identical borders', 'web-stories')}>
```

Comment:
Hm, the wording is a bit confusing indeed. Maybe it should be "Lock/unlock border width"?

Reply:
Funny, after thinking about it for some time, "Toggle identical borders" started sounding more understandable, too. We can still consult with UX though since ideally it would sound understandable right away without thinking about it 😛